### DISCUSSION TITLE
Custom Color Setting

### DISCUSSION THREAD

**Discussion by muazhari** [nomi-san/parsec-vdd#62] [Q&A] [answered]
Does this tool support customizing color settings? I see some notable color differences between Windows RDP and Parsec. It seems very unpleasant to me. If it doesn't, can you add it?

**Comment by nomi-san** [comment, marked as answer]
That's impossible because the Parsec VDD is closed source driver.
You can request this feature in some open source Iddcx driver project like 
https://github.com/itsmikethetech/Virtual-Display-Driver
